src/main/java/com/sigopt/model/TrainingMonitor.java

```diff
@@ -0,0 +1,34 @@
+package com.sigopt.model;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TrainingMonitor extends StructObject {
+    public TrainingMonitor() {
+        super();
+    }
+
+    public Integer getMaxCheckpoints() {
+        return Utils.asInteger(this.get("max_checkpoints"));
+    }
+
+    public List<APIObject> getEarlyStoppingCriteria() {
+        return Utils.mergeIntoList(new ArrayList<APIObject>(), this.get("early_stopping_criteria"), APIObject.class);
```

Comment:
shouldn't this have a more specific type?

Reply:
Since we will eventually have multiple types of stopping criteria trying to keep this as open as possible.